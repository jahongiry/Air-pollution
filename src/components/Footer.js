import classes from './Footer.module.css'

function Footer() {
  return <footer className={classes.footer}>
    <h4>Created by Jahongir Yusupov</h4>
  </footer>
}

export default Footer;