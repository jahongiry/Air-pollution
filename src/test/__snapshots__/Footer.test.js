import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Footer from '../../components/Footer';

it('Builds the snapchot of the footer component safely', () => {
  const three = renderer
    .create(
      <Router>
        <Footer />
      </Router>,
    )
    .toJSON();
  expect(three).toMatchSnapshot();
});
