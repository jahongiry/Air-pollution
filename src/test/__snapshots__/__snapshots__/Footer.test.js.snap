// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Builds the snapchot of the footer component safely 1`] = `
<footer
  className="footer"
>
  <h4>
    Created by Jahongir Yusupov
  </h4>
</footer>
`;
